pkg/webhook/admission/inferenceservice/inferenceservice_validator.go

```diff
@@ -43,9 +48,36 @@ func (validator *Validator) Handle(ctx context.Context, req admissiontypes.Reque
 		return admission.ErrorResponse(http.StatusInternalServerError, err)
 	}
 
+	if constants.IsEnableWebhookNamespaceSelector {
```

Comment:
alternatively, this could be done with a configmap value. For now, env var is probably fine.